pkg/grpc/grpc.go

```diff
@@ -92,8 +92,13 @@ func (g *Manager) GetGRPCConnection(address, id string, skipTLS, recreateIfExist
 			return nil, errors.Errorf("error generating x509 Key Pair: %s", err)
 		}
 
+		var serverName string
+		if id != "cluster.local" {
+			serverName = fmt.Sprintf("%s.default.svc.cluster.local", id)
```

Comment:
The `default` here must be configurable. right now this assumes that the target app is in the `default` namespace.

Reply:
fixed

Reply:
fixed